### PR TITLE
Return port message

### PR DESCRIPTION
**Pull request by dougperez**
- rename description to message 
- rename external_port_number to port_number
- fix RESOLUTION REQUIRED response was returning the code for message

### CODE REVIEW

File: lib/conduit/sprint/parsers/query_port_status.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def initialize(xml)
     def set_response_attributes
```
Comment (arvanasse) [inline lib/conduit/sprint/parsers/query_port_status.rb]: What do you think about breaking this method down into a series of smaller methods that each handle a particular message_type_code and using this method to delegate processing to them?

**Comment by captainbeardo** [comment]
:+1:

File: lib/conduit/sprint/parsers/query_port_status.rb (hunk 1 above)
Comment (dougperez) [inline lib/conduit/sprint/parsers/query_port_status.rb]: Yeah I agree this is kinda crazy method. Ill look into breaking it down.

**Comment by arvanasse** [comment]
:+1: Thanks, it's much easier to read

**Pull request merged**